### DISCUSSION TITLE
Question about target_node

### DISCUSSION THREAD

**Discussion by AIRCEKA** [Telmate/terraform-provider-proxmox#980] [General]
Hi guys,
I am sucessfully using the telmate provider for my 3 node proxmox cluster (with ceph) and i was curious about the option target_node.
Is there a way to have high availability in case the configured target node is down ? 

For the pm_api_url i use a failover dns so it only tries to reach available proxmox nodes.

Thanks for your help :)

**Comment by MrCurlsTTV** [comment]
I am also curious about this, I have 3 nodes as well for HA.
